code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -643,44 +645,64 @@ func (c *Chain) addInitialStakes(stakes []state.InitStake, creationDate common.T
 				v.ProviderType, v.ProviderID, v.ClientID)
 		}
 
-		sp.Pools[v.ClientID] = &stakepool.DelegatePool{
+		dp := &stakepool.DelegatePool{
 			Balance:      v.Tokens,
+			Reward:       0,
```

Comment:
Do not assign default zero value explicitly except the 0 value is special

Reply:
I was asked to follow code from `sp-lock` and reward has a 0 value assigned to it there. https://github.com/0chain/0chain/blob/c3f84ea91ff696d849bc7fb0d2ab9daa4dcb72f9/code/go/0chain.net/smartcontract/stakepool/lock.go#L63

Reply:
you can fix there also

Reply:
got it.